thingiverse/ThingiverseService.py

```diff
@@ -169,7 +207,9 @@ def getLiked(self) -> None:
         """
         if not self._checkUserNameConfigured():
             return
-        self._executeQuery("users/{}/likes".format(self._user_name))
+        self._clearSearchResults()
+        query = self._api_client.getLikesUrl()
+        self._executeQuery(query)
```

Comment:
Better to have this _clearSearchResults and is_from_collection logic in self._executeQuery? Now its in multiple places, making it hard to keep behavior up to date